doc/base/classes.xml

```diff
@@ -16068,17 +16068,21 @@
 			</return>
 			<argument index="0" name="host" type="String">
 			</argument>
+			<argument index="1" name="ip_type" type="int" default="int.IP_TYPE_ANY">
```

Comment:
I'm not sure about this default value, does doctool parse it this way or did you add it manually? If it's the latter, it will be changed next time we run doctool :)